Src/Contract/SwisProxyBase.cs

```diff
@@ -143,7 +141,7 @@ private void Initialize(EndpointAddress address, Binding binding, ServiceCredent
             SslStreamSecurityBindingElement element = elements.Find<SslStreamSecurityBindingElement>();
             if (element != null)
             {
-                element.IdentityVerifier = new SWIdentityVerifier();
+                //element.IdentityVerifier = new SWIdentityVerifier();
```

Comment:
What is the consequence of this being removed? (I was probably the one to remove it, but I don't remember why.)

Reply:
If uncommented, it produces the following error:

```bash
SwisProxyBase.cs(144,25,144,41): error CS1061: 'SslStreamSecurityBindingElement' does not contain a definition for 'IdentityVerifier' and no accessible extension method 'IdentityVerifier' accepting a first argument of type 'SslStreamSecurityBindingElement' could be found (are you missing a using directive or an assembly reference?)
```

This property seems to have been removed in the reduced surface area supported by .NET Standard and .NET Core.  I changed the target framework for **SolarWinds.InformationService.Contract.csproj** from **netstandard2.0** to **netcoreapp3.1** and got the same error.